packages/istanbul-lib-source-maps/test/transformer.test.js

```diff
@@ -47,9 +26,24 @@ function createData() {
         b: {}
     };
 
+function createData() {
```

Comment:
My intention with my comment was to move away from these one-shot factory functions in favour of having one base object that's extended.